Renaming buildtest help to buildtest show

Mendi03 opened the pull request:
Made the following updates:

- Switched the functionality of `buildtest help` to `buildtest show` in order to create new purpose for `buildtest help` command.
- Updated documentation
- **_test_help.py_** was changed to **_test_show.py_**
- Instances of `buildtest help` were changed to `buildtest show` in the **_show.py_** and _**test_show.py**_ file
- **_print_help_** methods changed to **_print_show_**

Mendi03 committed:
Changed buildtest help to buildtest show

The buildtest help command is now buildtest show.

Example:
The output for buildtest help build will now be the output buildtest show build

Mendi03 committed:
Renamed help.py file to show.py

Renamed "help.py" to "show.py" to leave the word "help" free for new feature

Mendi03 committed:
Deleted unecessary comment

Mendi03 committed:
Fixed Style check

Mendi03 committed:
fix for yamllint check

Mendi03 committed:
Updated buildtest help to buildtest show for documentation

kingxleyy reviewed (commented):
everything looks good to me

Mendi03 commented on tests/cli/test_help.py:
@shahzebsiddiqui should I change the name of the function `test_buildtest_help` to `test_buildtest_show`? Furthermore, would this mean I have to change `buildtest_help` to `buildtest_show`?

shahzebsiddiqui replied:
yes please change the name of method

Mendi03 committed:
changed the name of method buildtest_help to buildtest_show

changed the name and instances of the method test_buildtest_help to test_buildtest_show and changed buildtest_help to buildtest_show.
Latly, changed file name test_help to test_show.

Mendi03 committed:
Merge branch 'devel' into help_all

Mendi03 committed:
fixed import issue and call in main.py

Mendi03 committed:
Merge branch 'help_all' of https://github.com/Mendi03/buildtest into show

shahzebsiddiqui reviewed (approved):
LGTM. Great job @Mendi03
